docs/blueprints/README.md

```diff
@@ -37,22 +46,24 @@ However, it is expected that there should be feedback from the engineering team,
 
 ## Template
 
-The design document uses `TEMPLATE.md` as a guide. However, it can be modified when necessary.
+The blueprint uses `TEMPLATE.md` as a guide. However, it can be modified when necessary.
 
 The template has the following sections:
 
-* **Abstract** gives an overview of the design document and what it accomplishes. It does not go into details about the problem.
+* **Abstract** gives an overview of the blueprint and what it accomplishes. It does not go into details about the problem.
 
-* **Background** gives an overview of the background why this design document exists. It describes the problems the design document tries to solve. It describes the goals of the design.
+* **Background** gives an overview of the background why this blueprint exists. It describes the problems the blueprint tries to solve. It describes the goals of the design.
 
 * **Design** contains the solution and its parts. The level of detail should correspond to the level of risk. The more problems a wrong solution would cause, the more detailed should be the description. The design should describe the solution to the degree it is usable by the end-user.
 
 * **Rationale** section describes the alternate approaches and trade-offs. It should be clear why the proposed design was chosen among the alternate solutions.
 
-* **Implementation** describes the steps to complete this design document. It should contain a rough outline of tasks. If necessary, there can be additional details about the changes to the codebase and process.
+* **Implementation** describes the steps to complete this blueprint. It should contain a rough outline of tasks. If necessary, there can be additional details about the changes to the codebase and process.
+
+* **Wrap up** describes the steps needed
```

Comment:
it's 2 words or one word?

Here is 2 word however in the template is one word https://github.com/storj/storj/pull/2927/files#diff-25e11c5b04ae10c94492b0191cc65769R23

Reply:
i'm not sure it matters? it's fine to have both

Reply:
fine to me, I wanted to know if there was a typo in any of them.